R/runner.R

```diff
@@ -280,15 +272,7 @@ orderly_runner_ <- R6::R6Class(
     submit_workflow = function(reports, ref = NULL, changelog = NULL,
                                poll = 0.1, timeout = 60 * 60 * 3) {
       self$assert_ref_switching_allowed(ref)
-      if (!is.null(ref)) {
-        dependency_root <- self$alternative_root
```

Comment:
do we still need the alternative root at all with this change?

Reply:
Nope - no longer need it